packages/shared/src/hooks/useUserMention.ts

```diff
@@ -246,10 +246,10 @@ export function useUserMention({
     };
 
     const dom = commentRef.current?.getRootNode();
-    dom.addEventListener('mousedown', userClicked);
+    dom?.addEventListener('mousedown', userClicked);
```

Comment:
When would this be empty?

Reply:
Ok it is because of `strict:false` in tsconfig

Reply:
Oh interesting that it didn't fail before, I wonder why 🤯